ui/packages/shared/profile/src/ProfileSelector/index.tsx

```diff
@@ -201,13 +208,21 @@ const ProfileSelector = ({
               error={error}
             />
           </div>
-          <div className="w-full flex-1">
+          <div className="flex w-full flex-1">
             <MatchersInput
               queryClient={queryClient}
               setMatchersString={setMatchersString}
               runQuery={setQueryExpression}
               currentQuery={query}
             />
+            <div>
+              <Input
+                className="divider-y block w-full flex-1 rounded rounded-l-none bg-gray-50 px-2 py-2 text-sm outline-none focus:ring-indigo-800 dark:bg-gray-900"
+                placeholder="filter function name..."
```

Comment:
Placeholder should be capitalized to match the remainder of our placeholders/text

```suggestion
                placeholder="Filter function name..."
```